DRAFT_insectIndicators.Rmd

```diff
@@ -0,0 +1,1336 @@
+
+* Öberg, S., Gjershaug, J. O., Diserud, O. & Ødegaard, F. 2011. Videreutvikling av metodikk for 
+arealrepresentativ overvåking av dagsommerfugler og humler. Naturindeks for Norge. – NINA 
+Rapport 663. 
+
+
+## Analyses
+
+### Calculation principles for NorIns indicators
+The indicators from the Norwegian insect monitoring are calculated with functions in the `Norimon` R-package [link](https://github.com/NINAnor/Norimon). This methodology is meant to facilitate the calculation of a broad variety of insect indicators using data from **NorIns**.
+
+**Main steps of workflow:**
+
+1. Fetch data of biomass or community diversity from a centralized database. Diversity data can be filtered on several taxonomic levels, but biomass is only available for whole samples.
+2. Bootstrap observations and calculate mean and uncertainty of point estimates.
+3. Compare observations to reference points.
+4. Display and plot the results
+
+```{r flow_chart, out.width = '400pt', dev='png', echo=F}
+knitr::include_graphics("images/NorIns_flow_chart.png")
+
+```
+
+Most of the ecology comes into step 1, in choosing what data to use to describe a quality. This can be a custom selection of species, or a set of higher taxa such as specific genuses, families or even order. The goal is to choose a set of taxa that represent specific qualities of the community that can indicate the ecological status of the ecosystem. In addition to the selection of taxa, we have to decide on the level of aggregation, e.g. if we should aggregate the data on the region scale, locality scale, or even the individual sample occasion scale. For ecological indicators, we will aggregate the values to a year locality scale, where the variables summarises all catches in a locality in a year.[NOTE, implement 5 year/region aggregation]
+
+The second step is to get an estimate of the uncertainty of the data, were we use the bootstrap method. This method is flexible, as we could bootstrap the samples on different sampling levels (samples within localities, regions within country). For the insect indicators, we will bootstrap and summarize the variables to regional values, for example expressing the mean values of insect biomass in Trøndelag in 2021, with uncertainty.
+
+Ecological knowledge also comes into step 3, comparing the values to a reference state. Here we have several options. We could for example use a single defined value as the reference state. But we can also use a point in time as a reference point (e.g. the start of a time series).
+
+Taken together, this framework is meant to facilitate the calculation of an arbitrary set of insect indicators, based on the combination of choices in data to fetch, aggregation level, and reference comparison. We can work through an example with pollinators, to showcase the framework. 
+
+In the example, we start with connecting to the database in order to fetch the data.
+
+```{r}
+connect_to_insect_db()
```

Comment:
I get stuck here - 

Quitting from lines 275-276 [unnamed-chunk-16] (DRAFT_insectIndicators.Rmd)
Error:
! connection to server at "ninradardata01.nina.no" (10.1.4.92), port 5432 failed: connection to server at "ninradardata01.nina.no" (10.1.4.92), port 5432 failed: fe_sendauth: no password supplied

Reply:
Sending pwd through other channels...